src/apolloBootstrap.ts

```diff
@@ -0,0 +1,89 @@
+import { ApolloClient, InMemoryCache, gql } from '@apollo/client';
+import { InlineComment } from './lib/InlineComment';
+import { InlineSelection } from './lib/InlineSelection';
+
+
+export function apolloBootstrap(pageID: number) {
+    let refreshInlineComment: () => void;
+
+    const client = new ApolloClient({
+        uri: 'shared-draft-comment/graphql',
+        cache: new InMemoryCache(),
+    });
+
+    console.log(pageID);
+    const getSelections = () => client.query({
+        query: gql`
+            query getSelections($pageID: ID) {
+                readSelections(filter: {pageID: {eq: $pageID}}) {
+                    id
+                    endMeta {
+                        created
+                        textOffset
+                        parentTagName
+                        parentIndex
+                    }
+                    startMeta {
+                        created
+                        textOffset
+                        parentTagName
+                        parentIndex
+                    }
+                    text
+                }
+            }
+        `,
+        variables: { pageID },
+    }).then(result => {
+        return result.data.readSelections;
+    });
+
+    const postSelection = (selection: InlineSelection) => client.mutate({
```

Comment:
In a perfect world this could be done in one request but because DomMeta is its own DataObject, I have to create them first. Then assign them to a selection.